client/context.html

```diff
@@ -16,7 +16,13 @@
       {{> looping_video}}
     {{else}}
       <div class="image-container" style="height: {{mainStreamHeight}}px; width: {{leftSectionWidth}}px; max-width: {{overlayContentMaxWidth}}px;">
-        <img src="{{url}}" />
+        {{#if lgUrl}}
+          <img src="{{lgUrl}}" />
```

Comment:
I don't see an accessor method for `lgUrl`. Probably this should just be `url`, and then there should be `previewUrl`, which should be created in the `ImageBlock` model and then be used in the `preview_image_section`